consumer/dragonboard820c/guides/setup-development-environment.md

```diff
@@ -0,0 +1,537 @@
+Please read the **1.3.3 Connect to Wi-Fi** chapter of the [DB410c Major League Hack Quick Start Guide](https://developer.qualcomm.com/download/db410c/mlh-quick-start-guide.pdf) for more details about the ```nmtui``` and ```nmcli``` tools.
+After connecting to Wi-Fi get your IP addresses.
+
+```shell
+ip addr show
+```
+
+There are ways to setup static IP addresses on the board itself.
+I chose to use the DHCP Address Reservation feature of my router. I got the Ethernet and Wi-Fi MAC addresses and associated them with the desired IP addresses. This strategy suits me. I'd like to hear yours.
+
+## Install Base Packages
+This section will guide you through the process of setting up the base development environment to make sure your board has all the appropriate dependencies needed for development.
+
+```shell
+sudo apt update && \
+ sudo apt upgrade -y && \
+ sudo apt dist-upgrade -u -y && \
+ sudo apt-get clean && \
+ sudo apt autoremove -y && \
+ sudo apt-get install -y \
+  apt-utils \
+  ca-certificates \
+  lsb-release \
+  locales \
+  curl \
+  wget \
+  unzip \
+  procps \
+  tar \
+  openssh-server \
+  htop \
+  dos2unix \
+  nano \
+  mc \
+  tmux
+```
+
+## Install .NET Core 2.1 Remote Debugging
+Enable the remote debugger for .NET Core 2.1
+  
+```shell
+sudo apt-get install -y \
+ libunwind8 gettext apt-transport-https \
+ && curl -sSL https://aka.ms/getvsdbgshbeta | bash /dev/stdin -v latest -r linux-arm -l /vsdbg
+```
+
+## Install Docker dependencies
+Install Docker dependencies
+
+```shell
+sudo apt-get install -y \
+ gnupg2 software-properties-common
+```
+
+## Install Sensor Board dependencies
+Install Sensor Board dependencies
+
+```shell
+sudo apt-get install -y \
+ arduino-mk arduino git build-essential cmake bison doxygen graphviz python-sphinx python3-sphinx libudev-dev libgtest-dev libjson-c-dev autoconf libtool python-dev python-pip python3-pip pkg-config libpcre3-dev libjpeg-dev clang-6.0 libclang-6.0-dev python-clang-6.0 \
+ # && rm -rf /var/lib/apt/lists/* \
+```
+
+## Install verious other dependencies
+
+```shell
+sudo apt-get install -y python3-pip
+sudo apt-get install -y python3-dev python3-tk python3-numpy
+sudo apt-get install -y python3-scipy
+sudo apt-get install -y python-opencv
+sudo apt-get install -y libopencv-dev
+```
+
+Reboot
+
+```shell
+reboot
+```
+
+## Setup Desktop Environment
+If you installed the developer image you might want to add the Desktop Environment later or you installed the ALIP GUI image, but you want to use a different theme, perhaps another Desktop Manager than sddm.
+
+```shell
+sudo tasksel
+```
+
+This command provides a character based UI.
+Select the following by typing space.
+
+- Debian Desktop Environment
+- All or some indented GUI themes
+- SSH Server
+
+select sddm Desktop Manager when asked.
+  
+Reboot
+
+```shell
+reboot
+```
+
+## Setup linaro user
+
+```shell
+login linaro
+```
+
+Type linaro as a password.

```

Comment:
since you are doing a very high leve, generic user guide, it might be right at this point, to recommend changing the default password. especially because (iirc) the linaro user has full sudo access.

Reply:
Great idea. Added.